src/pymatgen/io/vasp/inputs.py

```diff
@@ -1250,6 +1250,8 @@ def automatic(cls, subdivisions: int) -> Self:
         Returns:
             Kpoints object
         """
+        warnings.warn("Please use INCAR KSPACING tag.", DeprecationWarning, stacklevel=2)
```

Comment:
Not sure if we should follow [VASP Wiki's recommendation](https://www.vasp.at/wiki/index.php/KPOINTS):
> Deprecated: The [KSPACING](https://www.vasp.at/wiki/index.php/KSPACING) tag provides almost the same functionality. Preferably use that method instead.

Reply:
To be backward compatible, this needs to stay in the input sets

Reply:
Thanks a lot for the comment! I'm not sure if `pymatgen` wants to broadcast this warning at all in the first place, because it's more a VASP thing.

> this needs to stay in the input sets.

I don't fully understand sorry. My intention was to emit a warning when `Kpoints.automatic()` is called (which agrees with the VASP manual), why should this be inside the input set (`VaspInputSet`?)?

Reply:
@JaGeo Can you perhaps elaborate on this comment please? Sorry I don't fully understand.

Reply:
Sorry! i misunderstood something here! I don't have any further comments.

Reply:
All good! I would mark it as resolved then. Do feel free to comment if you have any!